templates/partials/person_jobs_card.html

```diff
@@ -0,0 +1,32 @@
+<div class="card-body">
+  <h3 class="card-title mb-4">Employment history for {{ entity }}</h3>
+  <h4 class="card-subtitle">Over the last ##### years...</h4>
+  <div style="min-height: 300px; background-color: lavender"></div>
```

Comment:
For consistency's sake...

```suggestion
  <div style="min-height: 300px; background-color: lavender" class="col-lg-12"></div>
```